app/services/spree_avatax_official/transactions/create_service.rb

```diff
@@ -42,6 +46,20 @@ def send_request(order, transaction_type, options)
 
         client.create_transaction(create_transaction_model, options)
       end
+
+      def log_unusual_order(order, response)
+        Airbrake.notify('[AVATAX] Unusual order with total higher than 100k:', {
+          order: order.inspect,
+          adjustments: order.adjustments,
```

Comment:
Also, how about changing this line `order.adjustments.map(&:inspect).join(', ')` or something like that? I think by default the `to_s` conversion can limit the amount of items that will get logged

Reply:
Good idea, I will change that!

Reply:
Added